packages/terra-core-docs/src/terra-dev-site/doc/show-hide/example/InitiallyOpenShowHide.jsx

```diff
@@ -20,8 +21,11 @@ class InitiallyOpenShowHide extends React.Component {
 
   render() {
     return (
-      <ShowHide preview={<p>{previewText}</p>} isOpen={this.state.isOpen} onChange={this.toggleShowHide}>
-        <p>{fullText}</p>
+      <ShowHide focusRef={this.focusRef} preview={<p>{previewText}</p>} isOpen={this.state.isOpen} onChange={this.toggleShowHide}>
+        <p>
+          { previewText }
+          <span ref={this.focusRef}>{ text }</span>
```

Comment:
is this intended change to use same ref for two different html elements.

Reply:
The `<span>` node created on line 27 needs to be accessed in the `<ShowHide>` component in order to receive focus, so `<ShowHide>` is taking a focusRef prop, which will the reference to the `<span>`. 

I understand that it's confusing as I don't use ref forwarding. The reason for that is following:
- The ref variable and `span` node and both created at the same component;
- The prop can have meaningful name, like `focusRef`;
- The prop is not required.
But I am open for suggestions.